src/hammer-vlsi/defaults.yml

```diff
@@ -794,6 +808,28 @@ sim.inputs:
   # If set to True, the sim will be time annotated based on synthesis or PAR results.
   timing_annotated: false
 
+  # Settings related to activity file (SAIF) dumping
+  saif:
+    # Enable dumping an SAIF file, with activity factors
+    enable: false
+    # How Hammer determine when to start or stop the activity dumping
+    # Valid options:
+    # time - Start dumping at the specified start_time and end at the specified end_time
+    # trigger - Inserts the trigger into the simulators run script before starting the dump and
+    #   after enabling dumping before stopping dumping. See your simulator plugin for an example
+    # none - Start dumping at the beginning of simulation and stop at the end.
+    # type: str
+    mode: "none"
+    # The times the simulator should begin and end its dumping of the SAIF file.
+    # type: TimeValue
+    start_time: "0ns"
+    # type: TimeValue
+    end_time: "1ns"
+    # The triggers that will occur when the dump should start and when the dump should end.
```

Comment:
What are some examples of this setting?

Reply:
I think we should have a general discussion about this setting to see what level the API should be at.
It is currently just the raw commands and so I think the documentation would have to be in the tool plugins, but I did a little research into what we might be able to support on commercial simulators so we potentially could have a higher level API.

Reply:
Updated.